src/config.js

```diff
@@ -29,39 +29,31 @@ let cfgConfig = {};
 
 // Check if there is a user provided config file
 let cfgFile = path.resolve(nconf.get('configfile') || './.config.js');
-let cfgReason = '';
-const cfgFileExt = cfgFile.substr((cfgFile.lastIndexOf('.') + 1));
-if (cfgFile && fs.existsSync(cfgFile)) {
-  const cfgRead = fs.readFileSync(cfgFile, { encoding: 'utf-8' });
-  logger.info(`timings API config [${cfgFile}]:\n\n${cfgRead}\n`);
-  switch (cfgFileExt) {
-    case 'js':
-      cfgConfig = require(cfgFile);
-      break;
-    case 'json':
-      try {
+const cfgRead = fs.existsSync(cfgFile) ? fs.readFileSync(cfgFile, { encoding: 'utf-8' }) : '';
+if (cfgRead) {
+  const cfgFileExt = cfgFile.substr((cfgFile.lastIndexOf('.') + 1));
+  try {
+    switch (cfgFileExt) {
+      case 'js':
+        cfgConfig = require(cfgFile);
+        break;
+      case 'json':
         cfgConfig = JSON.parse(cfgRead);
-      } catch (err) {
-        cfgReason = err.message;
-      }
-      break;
-    case 'yaml':
-    case 'yml':
-      try {
+        break;
+      case 'yaml':
+      case 'yml':
         cfgConfig = yaml.safeLoad(cfgRead);
-      } catch (err) {
-        cfgReason = `${err.name} - ${err.reason}`;
+        break;
+      default:
         cfgFile = `defaults`;
-      }
-      break;
-    default:
-      cfgReason = `Unknown file type [${cfgFileExt.toUpperCase()}]`;
-      cfgFile = `defaults`;
+    }
+  } catch (err) {
+    cfgFile = `defaults`;
```

Comment:
If there's an error, I'd think you'd want to let the bubble up (and terminate with non-zero status code).  Why are you falling back to this?

Also, you don't need backticks here.